certified-connectors/IN-D Payables/apiDefinition.swagger.json

```diff
@@ -1,3 +1,789 @@
+                              }
+                            },
+                            "description": "Each item corresponds to the values extracted in a row against the Header items (Validated)"
+                          }
+                        },
+                        "description": "The line items in the Table is extracted (Validated)"
+                      }
+                    }
+                  },
+                  "description": "The Result Data for the whole batch of invoices (Validated)"
+                }
+              }
+            }
+          },
+          "403": {
+            "description": "Authorization headers or API key not given",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Authorization headers or API key not given",
+                  "title": "Authorization error"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Invalid Request",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Invalid Request",
+                  "title": "Invalid Request"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal error",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Internal error",
+                  "title": "Internal error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Export validated data",
+        "description": "Export validated invoice data using particular batch id",
+        "operationId": "ExportValidatedData",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "batch_id": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "validated invoice batch data",
+                  "title": "batch id"
+                }
+              },
+              "required": [
+                "batch_id"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.in-d.ai"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.in-d.ai/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;IT Operations"
+    }
+  ],
+  "definitions": {},
 {
```

Comment:
@gowthamd3 Swagger file is duplicated below: Please remove this..

Reply:
Removed duplicate file @sriyen-msft